File: Clients/Java DB Manager/src/ctsbdb/DBman.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ctsbdb;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.ResultSetMetaData;
import java.sql.SQLException;
import java.sql.Statement;

/**
 *
 * @author Vinay
 */
public class DBman {

    private String myUsername = "";
    private String myPassword = "";
    private String url;

    public DBman(String username, String password) {
        myUsername = username;
        myPassword = password;
        url = "jdbc:sqlserver://ctsb.database.windows.net:1433;database=CPscores;user=" + myUsername + "@ctsb;password=" + myPassword + ";encrypt=true;trustServerCertificate=true;hostNameInCertificate=*.database.windows.net;loginTimeout=30;";
    }

    public void printTable() {
        System.out.print("Attempting connection...");
        try {
            Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
            Connection conn = DriverManager.getConnection(url);
            System.out.println(" Conected!\n");
            Statement st = conn.createStatement();
            ResultSet rs = st.executeQuery("SELECT * FROM TeamScores");
            ResultSetMetaData rsmd = rs.getMetaData();
            int columnsNumber = rsmd.getColumnCount();

            while (rs.next()) {
                for (int i = 1; i <= columnsNumber; i++) {
                    System.out.printf("%-15s", rs.getString(i));
                }
                System.out.println();
            }
        } catch (ClassNotFoundException | SQLException e) {
            System.out.println(" Error!");
            System.out.println(e);
        }
    }

    public void clearTable() {
        System.out.print("Attempting connection...");
        try {
            System.out.println(" Connected!");
            Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
            Connection conn = DriverManager.getConnection(url);
            System.out.println("Conected");
            Statement st = conn.createStatement();
            System.out.println("Clearing table...");
            try {
                st.executeQuery("Truncate table TeamScores");
            } catch (SQLException e) {
            }
            System.out.println("Table cleared");
        } catch (ClassNotFoundException | SQLException e) {
            System.out.println(" Error!");
            System.out.println(e);
        }
    }

    public void add(String teamname, int score, int time) {
        System.out.print("Attempting connection...");
        try {
            Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
            Connection conn = DriverManager.getConnection(url);
            System.out.println(" Conected");

            String sql = "INSERT INTO TeamScores(TeamName,Score,Time) VALUES(?,?,?)";
            PreparedStatement pstmt = conn.prepareStatement(sql);
            System.out.print("Sending data...");
            pstmt.setString(1, teamname);
            pstmt.setInt(2, score);
            pstmt.setInt(3, time);
            pstmt.executeUpdate();
            System.out.println(" Success!");
        } catch (ClassNotFoundException | SQLException e) {
            System.out.println(" Error!");
            System.out.println(e);
        }
    }

    public String connect() {
        try {
            Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
            Connection conn = DriverManager.getConnection(url);
            return "good";
        } catch (ClassNotFoundException | SQLException e) {
            return e.toString();
        }
    }

    public void setUsername(String username) {
        myUsername = username;
        url = "jdbc:sqlserver://ctsb.database.windows.net:1433;database=CPscores;user=" + myUsername + "@ctsb;password=" + myPassword + ";encrypt=true;trustServerCertificate=true;hostNameInCertificate=*.database.windows.net;loginTimeout=30;";
    }

    public void setPassword(String password) {
        myPassword = password;
        url = "jdbc:sqlserver://ctsb.database.windows.net:1433;database=CPscores;user=" + myUsername + "@ctsb;password=" + myPassword + ";encrypt=true;trustServerCertificate=true;hostNameInCertificate=*.database.windows.net;loginTimeout=30;";
    }

    public String info() {
        return url.substring(0, url.indexOf("database="));
    }
}

File: Clients/CSDBman/CSDBman/DBman.cs
using System;
using System.Data.Odbc;
using DT = System.Data;
using QC = System.Data.SqlClient;

namespace CSDBman
{
    class DBman
    {
        private String myUsername;
        private String myPassword;
        private String myUrl;
        private String myDB;
        private String myServer;

        public DBman(String username, String password, String server, String db)
        {
            myUsername = username;
            myPassword = password;
            myDB = db;
            myServer = server;
            myUrl = "Server = tcp:" + server + ",1433; Initial Catalog = + " + db + "; Persist Security Info = False; User ID = " + username + "; Password =" + password + "; MultipleActiveResultSets = False; Encrypt = True; TrustServerCertificate = False; Connection Timeout = 30;";
        }

        public void printTable()
        {
            string cmd = "SELECT * FROM TeamScores";
            Console.Write("Attempting connection...");
            using (QC.SqlConnection conn = new QC.SqlConnection(myUrl))
            {
                using (QC.SqlCommand command = new QC.SqlCommand(cmd))
                {
                    try
                    {
                        conn.Open();
                        Console.WriteLine(" Conected!\n");
                        DT.DataTable Table = new DT.DataTable(myDB);
                        foreach (DT.DataRow dataRow in Table.Rows)
                        {
                            foreach (var item in dataRow.ItemArray)
                            {
                                Console.WriteLine(item);
                            }
                        }
                    }
                    catch (QC.SqlException e)
                    {
                        Console.WriteLine("An error has occured");
                        Console.WriteLine(e);
                    }

                }
            }
        }

        public void clearTable()
        {
            string cmd = "Truncate table TeamScores";
            Console.WriteLine("Attempting connection...");

            using (QC.SqlConnection conn = new QC.SqlConnection(myUrl))
            {
                using (QC.SqlCommand command = new QC.SqlCommand(cmd))
                {
                    command.Connection = conn;
                    try
                    {
                        Console.WriteLine("Opening connection...");
                        conn.Open();
                        Console.Write("Clearing data... ");
                        command.ExecuteNonQuery();
                        Console.WriteLine(" Done!");
                    }
                    catch (Exception e) { }
                }
            }
        }

    public void setUsername(String username)
        {
            myUsername = username;
            myUrl = "Server = tcp:" + myServer + ",1433; Initial Catalog = + " + myDB + "; Persist Security Info = False; User ID = " + myUsername + "; Password =" + myPassword + "; MultipleActiveResultSets = False; Encrypt = True; TrustServerCertificate = False; Connection Timeout = 30;";
        }

        public void setPassword(String password)
        {
            myPassword = password;
            myUrl = "Server = tcp:" + myServer + ",1433; Initial Catalog = + " + myDB + "; Persist Security Info = False; User ID = " + myUsername + "; Password =" + myPassword + "; MultipleActiveResultSets = False; Encrypt = True; TrustServerCertificate = False; Connection Timeout = 30;";
        }

        public String info()
        {
            return myUrl.Substring(0, myUrl.IndexOf("database="));
        }
    }
}